src/TagsInput.vue

```diff
@@ -41,6 +41,7 @@
 <script>
 export default {
     props: {
+        max: number,
```

Comment:
Let's call the prop "limit". Also, the prop type should be capitalized, i.e. "limit: Number"